packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -86,6 +87,7 @@ const _ActionList = ({ children, surfaceLevel = 2, testID }: ActionListProps): J
   } = useDropdown();
 
   const { theme } = useTheme();
+  const { isInBottomSheet } = useBottomSheetContext();
```

Comment:
Nit: we can call this in child components only no rather than calling here and passing as props to child?